components/brave_shields/common/features.cc

```diff
@@ -10,6 +10,10 @@
 namespace brave_shields {
 namespace features {
 
+// When disabled, Brave will allow first-party requests in "standard" blocking
```

Comment:
I would rather rephrase it as "When enabled ..." :)

Reply:
I think the "disabled" state is less obvious behavior to those unfamiliar with adblocking - I added a description for both.